README.md

```diff
@@ -84,15 +173,15 @@ This date format is used all over the web, and puts each field in consistent ord
 
 ### API Keys
 
-These standards do not take a position on whether or not to use API keys.
+We recommend using [api.data.gov](https://api.data.gov/about/) as a layer between your API and users. This API Management service provides:
+* API key management
+* rate limiting (throttling)
+* gathering usage statistics (analaytics)
 
-But _if_ keys are used to manage and authenticate API access, the API should allow some sort of unauthenticated access, without keys.
+Keys managed by api.data.gov can be re-used with other APIs hosted by this service, which reduces complexity for users. This service also allows the use of a DEMO_KEY for unauthenticated access, without keys. (And example of this is the [FBOpen API](https://pages.18f.gov/fbopen/apidocs/).)
```

Comment:
From Cindy S: text contained within parens should state (An….) not And.
3rd bullet the word analytics is mis-spelled – remove the a after the l.